jdbc/src/main/java/nextstep/jdbc/JdbcTemplate.java

```diff
@@ -14,4 +21,73 @@ public class JdbcTemplate {
     public JdbcTemplate(final DataSource dataSource) {
         this.dataSource = dataSource;
     }
+
+    public void update(final String sql, final Object... objects) {
+        executePreparedStatement(sql, pstmt -> {
+            setParams(pstmt, objects);
+            pstmt.executeUpdate();
+            return null;
```

Comment:
```suggestion
        executePreparedStatement(sql, pstmt -> {
            setParams(pstmt, objects);
            return pstmt.executeUpdate();
```

이프의 의견을 들어보고싶어요! 저는 null을 반환하는걸 선호하지 않는 편이라 만약 제가 코드 짰다면 위처럼 return해줬을 것 같은데, 이프는 어떻게 생각하실지 궁금해요! 아무래도 사용하지 않는 값을 반환하게 되는 것 같다는 생각도 들어서 이프의 생각도 들어보고싶어요!

Reply:
와!!! 이거 고민많았는데!! 좋은 생각 감사합니다~!!! 바로 적용했어요ㅋㅎㅎ